test/test_nestedtensor.py

```diff
@@ -559,9 +559,12 @@ def test_nested_namespace(self):
         nested_namespace_result = torch.nested.to_padded_tensor(nt, 4)
         self.assertEqual(result, nested_namespace_result)
 
-    def test_to(self):
+    @parametrize("layout", [torch.strided, torch.jagged], name_fn=layout_name)
```

Comment:
To cover the CUDA cases, we should move this test over to the device-specific area. Either somewhere under here:
https://github.com/pytorch/pytorch/blob/dd00f5e78d44f55595728ef03018c01296a31ec9/test/test_nestedtensor.py#L3464

or here:
https://github.com/pytorch/pytorch/blob/dd00f5e78d44f55595728ef03018c01296a31ec9/test/test_nestedtensor.py#L828